src/source.ts

```diff
@@ -261,7 +297,7 @@ abstract class SourceContractItem implements Linkable {
 
   @memoize
   get natspec(): NatSpec {
-    if (this.astNode.documentation === null || this.astNode.documentation === undefined) {
+    if (!('documentation' in this.astNode) || this.astNode.documentation === null || this.astNode.documentation === undefined) {
```

Comment:
This should not be necessary. We should add the `documentation` member in the new interfaces.